java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java

```diff
@@ -0,0 +1,29 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Supports commands and transactions for the "Bitmap Commands" group for standalone and cluster
+ * clients.
+ *
+ * @see <a href="https://redis.io/docs/latest/commands/?group=bitmap">Bitmap Commands</a>
+ */
+public interface BitmapBaseCommands {
+    /**
+     * Returns the bit value at <code>offset</code> in the string value stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/getbit/">redis.io</a> for details.
+     * @param key The key for the string to get the bit at <code>offset</code> of.
+     * @param offset The index of the bit to return.
+     * @return The bit at offset of the string. Returns zero if the key is missing as it is treated as
```

Comment:
```suggestion
     * @return The bit at offset of the string. Returns zero if the key is empty or if the positive <code>offset</code> exceeds the length of the string.
```
Don't need to explain the reasoning behind the return.

Reply:
Consider: throws an error if offset is negative.